third_party/closure-compiler/runner/src/AmpCommandLineRunner.java

```diff
@@ -0,0 +1,34 @@
+import com.google.common.collect.ImmutableSet;
+import com.google.javascript.jscomp.CommandLineRunner;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.javascript.jscomp.CustomPassExecutionTime;
+
+public class AmpCommandLineRunner extends CommandLineRunner {
+  
+  ImmutableSet<String> suffixTypes = ImmutableSet.of(
+      "log.dev.fine",
+      "log.dev.assert"
+      );
+
+  protected AmpCommandLineRunner(String[] args) {
+    super(args);
+  }
+
+  @Override
+  protected CompilerOptions createOptions() {
+    CompilerOptions options = super.createOptions();
+    AmpPass ampPass = new AmpPass(getCompiler(), suffixTypes);
+    options.addCustomPass(CustomPassExecutionTime.BEFORE_OPTIMIZATIONS, ampPass);
```

Comment:
@cramforce there is a way to get `stripPrefix` to work and that is if we assign it to the global "AMP" namespace and use it like `AMP.dev.fine("text")` since that is never mangled or renamed it can strip it. this would mean though that the `log` module would need to be the initialized even earlier as it might be needed right away.